fin-advisor-crm/static/js/graphs.js

```diff
@@ -1,14 +1,37 @@
-function loadChart(elementId, endPoint) {
+let chart = null;
+function loadChart(
+    selectedTicker, 
+    chartendPoint, 
+    chartElementId,
+    selectedDate,
+    ticker_tag,
+    date_tag,
+    ) 
+    {
+    const endPoint = chartendPoint;
     console.log("Fetching data from "+ endPoint);
-    fetch(endPoint)
+    console.log("Selected ticker "+ selectedTicker + "for date" + selectedDate);
+
+    const payload = {
+        // 'equityticker' : selectedTicker,
+        [ticker_tag] : selectedTicker,
+        [date_tag] : selectedDate
+    }
+
+    fetch(endPoint, {
+        method: "POST",
+        body: JSON.stringify(payload),
+        headers: { 'Content-Type': 'application/json' }        
+    })
```

Comment:
`graphs.js` is meant to be used by all pages in the application so since this is very specific to your page put it in a different .js file and call that in your html file.

Reply:
This function is a generic function that can be implemented by any app within the project.